public/app/features/dashboard-scene/settings/variables/VariableEditorList.tsx

```diff
@@ -71,6 +74,10 @@ export function VariableEditorList({
             {(provided) => (
               <tbody ref={provided.innerRef} {...provided.droppableProps}>
                 {variables.map((variableScene, index) => {
+                  if (!isVariableEditable(variableScene)) {
```

Comment:
Is this needed if you do filtering at line 50?

Reply:
@tskarhed should not be, if line 76 uses editableVariables

Reply:
@tskarhed no, we need this here as we cannot use editableVariables as then the indices will be wrong

Reply:
Maybe we can add a small comment mentioning that